client/src/pages/announcement/news_notice/AnnouncementTable.tsx

```diff
@@ -30,12 +33,19 @@ const UnstyledAnnouncementTable: FunctionComponent<AnnouncementTableProps> = ({
 }) => {
   const history = useHistory();
   const [page, setPage] = React.useState(0);
-  const [rowsPerPage] = React.useState(isMobile? 5 : 10);
+  const [rowsPerPage] = React.useState(isMobile ? 5 : 10);
+
+  useEffect(() => {
+    if (!selectedID) {
```

Comment:
I added this section. 
@jjaykim 
Basically if even when no ids were selected, your section for setting pagenum ran and caused no data to be put into the table. 
There are numourous bugs other than this. Please fix it.

Reply:
and you were using useMemo.
useMemo should be used when getting a value.